.github/config/linux.json

```diff
@@ -14,7 +14,7 @@
         "gin_unittests",
         "blink_unittests",
         "content_shell",
-        "starboard/libstarboard.so.17"
+        "cobalt:gn_all"
```

Comment:
I think I forgot to add the same target to the android json config, if you're able to do that and it just works that'd be super helpful (or if it fails you can back it out and we can just submit this)

Reply:
Doing it in a different PR: https://github.com/youtube/cobalt/pull/4302

That PR has a better approach than this one IMO

Reply:
I could have added the other one in the same PR but i wasn't sure if builds will fail or what so created a new one.
Next time will try to reuse the same PR .